shuffle/cli/src/account.rs

```diff
@@ -174,6 +180,28 @@ pub fn create_account_onchain(
     Ok(())
 }
 
+async fn create_account_via_faucet(network: &Network, account: &LocalAccount) -> Result<()> {
+    println!("Creating a new account on {}...", network.get_name());
+    let faucet_client = FaucetClient::new(
+        network.get_faucet_url().to_string(),
+        network.get_json_rpc_url().to_string(),
+    );
+
+    let auth_key = account.authentication_key();
+    tokio::task::spawn_blocking(move || {
+        faucet_client.create_account(auth_key, "XUS").unwrap();
```

Comment:
Why does this need to be in a `tokio::tas::spawn_blocking`? Can't you just invoke it since you're in an `async` method already? And why unwrap as opposed to `?`?

Reply:
I did a deep dive on my own, and this is great as is. I misunderstood that `faucet_client.create_account(auth_key, "XUS")` is in fact not async and would block the event loop w a network call.

resolving.

Reply:
Based on the examples, they still use `?` over `unwrap` so let's clean that up as best we can.

https://docs.rs/tokio/0.2.22/tokio/task/fn.spawn_blocking.html#examples

```rust
use tokio::task;

let res = task::spawn_blocking(move || {
    // do some compute-heavy work or call synchronous code
    "done computing"
}).await?;

assert_eq!(res, "done computing");
```

Reply:
got it!